Library/Homebrew/github_packages.rb

```diff
@@ -225,21 +240,25 @@ def upload_bottle(user, token, skopeo, formula_name, bottle_hash)
         digest:      "sha256:#{manifest_json_sha256}",
         size:        manifest_json_size,
         platform:    platform_hash,
-        annotations: {},
+        annotations: { "org.opencontainers.image.ref.name" => tag },
       }
     end
+    formula_annotations_hash["org.opencontainers.image.created"] = created_times.max.strftime("%F")
```

Comment:
```suggestion
    formula_annotations_hash["org.opencontainers.image.created"] = Time.now.strftime("%F")
```

This shouldn't use the `tab.source_modified_time` because that's from the upstream source tarball date. I'd also recommend just adding it inline with the other `formula_annotations_hash` definitions.